runtime/Python2/src/antlr4/InputStream.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, data):
 
     def _loadString(self):
         self._index = 0
-        self.data = [ord(c) for c in self.strdata]
+        self.data = list(from_unicode(self.strdata))
```

Comment:
Actually, `from_unicode` returns a generator, not a list.

Since existing clients of `self.data` expect it to be a list, this eagerly evaluates the generator and converts the resulting items to a list.